File: Python/Car.py
#import library
from Vehicle import Vehicle

#deklarasi class
class Car(Vehicle):

    #private atribut
    __jmlKursi = ""
    __jmlPintu = ""

    #constructor
    def __init__(self, jmlKursi = "", jmlPintu = ""):
        self.__jmlKursi = jmlKursi
        self.__jmlPintu = jmlPintu
    
    #setter
    def setjmlKursi(self, jmlKursi):
        self.__jmlKursi = jmlKursi
    def setjmlPintu(self, jmlPintu):
        self.__jmlPintu = jmlPintu
     
    #getter
    def getjmlKursi(self):
        return self.__jmlKursi
    def getjmlPintu(self):
        return self.__jmlPintu
    

File: C++/Car.cpp
#pragma once

#include <iostream>
#include <string.h>
#include "Vehicle.cpp"

using namespace std;

class Car : public Vehicle
{
    // atribut private
private:
    string kursi;
    string pintu;

public:
    // constructor
    Car()
    {
    }
    Car(string kursi, string pintu, string nomor, string merk, string tahun, string warna) : Vehicle(nomor, merk, tahun, warna)
    {
        this->kursi = kursi;
        this->pintu = pintu;
    }

    // setter
    void setKursi(string kursi)
    {
        this->kursi = kursi;
    }
    void setPintu(string pintu)
    {
        this->pintu = pintu;
    }

    // getter
    string getKursi()
    {
        return this->kursi;
    }
    string getPintu()
    {
        return this->pintu;
    }

    ~Car()
    {
    }
};